packages/codemod/src/lib/upgrade.ts

```diff
@@ -0,0 +1,42 @@
+import fs from 'fs';
+import path from 'path';
+import semver from 'semver';
+import { transform } from './transform';
+import { TransformOptions } from './transform-options';
+
+const bundle = [
+  'remove-ai-stream-methods-from-stream-text-result',
+  'remove-experimental-ai-fn-exports',
+  'replace-baseurl',
+  'replace-nanoid',
+  'rewrite-framework-imports',
```

Comment:
what does this particular migration do? (`rewrite-framework-imports`)

Reply:
It rewrites for [this](https://sdk.vercel.ai/docs/troubleshooting/migration-guide/migration-guide-4-0#removed-svelte-vue-and-solidjs-exports), across any of svelte/vue/solid:

```
import { useChat } from 'ai/solid';
```

to

```
import { useChat } from '@ai-sdk/solid';
```